harvard-stellenbosch-university.csl

```diff
@@ -63,7 +63,7 @@
   </macro>
   <macro name="author-short">
     <names variable="author">
-      <name form="short" and="symbol" delimiter-precedes-last="never" et-al-min="6" et-al-use-first="5" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize-with="." sort-separator=" "/>
+      <name form="short" and="symbol" delimiter-precedes-last="never" et-al-min="10" et-al-use-first="5" et-al-subsequent-min="3" et-al-subsequent-use-first="1" initialize-with="." sort-separator=" "/>
```

Comment:
@Yusufras, what did you base this on?

I could only find:

> For first reference in text:
Coherence is an aspect of discourse competency that refers to the sequence of continuity of ideas in written or spoken texts (Davies, Brown, Elder, Hill, Lumley & McNamara, 1999:25).

> For all subsequent references in text:
Coherence is an aspect of discourse competency that refers to the sequence of continuity of ideas in written or spoken texts (Davies et al., 1999:25).

at http://libguides.sun.ac.za/content.php?pid=344501&sid=2970868